kratos/linear_solvers/amgcl_solver.h

```diff
@@ -78,7 +78,7 @@ enum AMGCLIterativeSolverType
 
 enum AMGCLCoarseningType
 {
-    RUGE_STUBEN,AGGREGATION,SA,SA_EMIN
+    RUGE_STUBEN,AGGREGATION,SA,SA_EMIN,TENTATIVE_PROLONGATION,POINTWISE_AGGREGATES,PLAIN_AGGREGATES
```

Comment:
These (`TENTATIVE_PROLONGATION,POINTWISE_AGGREGATES,PLAIN_AGGREGATES`) are not coarsening types, but just helper classes; I don't think they belong her.

Reply:
OK, I was testing, I thoug I was doing some mistake, reverting